src/saltext/azurerm/utils/azurerm.py

```diff
@@ -332,19 +338,22 @@ def get_identity_credentials(**kwargs):
     for keyword, value in kwarg_map.items():
         if kwargs.get(keyword):
             os.environ[value] = kwargs[keyword]
-
     try:
         if kwargs.get("cloud_environment") and kwargs.get("cloud_environment").startswith("http"):
             authority = kwargs["cloud_environment"]
         else:
             authority = getattr(
                 KnownAuthorities, kwargs.get("cloud_environment", "AZURE_PUBLIC_CLOUD")
             )
-        log.debug("AUTHORITY: %s", authority)
```

Comment:
**question (blocking):** why remove this log message?

Reply:
Oops, thought it was a debug statement of mine. Added back.